config/dev.exs

```diff
@@ -62,3 +62,9 @@ config :cbus_elixir, CbusElixir.Repo,
   database: "cbus_elixir_dev",
   hostname: "localhost",
   pool_size: 10
+
+config :cbus_elixir,
```

Comment:
Is it better to put this in `config.ex` (move from dev and test files) so it will be the default for dev and test and then move what is in `config.ex` using the env vars over to the `prod.ex` file?

Reply:
No because we don't want to accidentally load the dummy passwords in prod (if prod.exs ever got modified/removed)